src/content/fr/2019/media.md

```diff
@@ -0,0 +1,416 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=284657706&format=interactive"
+  )
+}}
+
+L’utilisation de la balise `<meta>` en HTML pour énoncer les Indications Client est beaucoup plus courante que l’en-tête HTTP. Cela reflète probablement la facilité avec laquelle on peut modifier les gabarits HTML par rapport à l’ajout d’en-têtes HTTP dans les outils intermédiaires. Cependant, si l’on considère l’utilisation de l’en-tête HTTP, plus de 50&nbsp;% de ces cas proviennent d’une seule plate-forme SaaS (Mercado).
+
+En observant la manière dont les Indications Client sont énoncées, on peut voir que la majorité des pages les utilisent pour les trois cas d’utilisation originaux&nbsp;: `DPR`, `ViewportWidth` et `Width`. Bien sûr, l’Indication Client `Width` nécessite l’utilisation de `<img sizes>` pour que le navigateur ait suffisamment de contexte sur la mise en page. 
+
+{{ figure_markup(
+  image="fig18_enabled_client_hints.png",
+  caption="Indications Client actives.",
+  description="Un graphique en anneau montrant que 26,1&nbsp;% des Indications Client utilisent `dpr`, 24,3&nbsp;% `viewport-width`, 19,7&nbsp;% `width`, 6,7&nbsp;% `save-data`, 6,1&nbsp;% `device memory`, 6,0&nbsp;% `downlink`, 5,6&nbsp;% `rtt` et 5,6&nbsp;% `ect`.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1878506264&format=interactive"
+  )
+}}
+
+Les Indications Client liés au réseau, `downlink`, `rtt`, et `ect`, ne sont disponibles que sur Chrome pour Android.
+
+### Lazy loading
+
+L’amélioration des performances des pages web peut être partiellement caractérisée comme un jeu de miroirs&nbsp;: sans les supprimer, on décale les éléments les plus lents en dehors de la zone d’usage de l’utilisateur. Par exemple, le <span lang="en">lazy loading</span> d’images est une de ces illusions où les images et les contenus médias ne sont chargés que lorsque l’utilisateur fait défiler la page. Cela améliore les performances perçues, même sur des réseaux lents, et évite à l’utilisateur de télécharger des octets qui ne sont pas visualisés autrement.
+
+Plus tôt, dans la <a href="#fig-5">Figure 5</a>, nous avons montré que le volume de contenu d’image au 75e percentile est bien plus important que ce qui pourrait théoriquement être affiché sur un seul écran de bureau ou de téléphone portable. L’audit <span lang="en">Lighthouse</span> concernant les [images hors champs](https://developers.google.com/web/tools/lighthouse/audits/offscreen-images) confirme ce soupçon. La page web médiane contient 27&nbsp;% d’images situées nettement sous la partie visible de la page. Ce pourcentage passe à 84&nbsp;% au 90e percentile. 
+
+{{ figure_markup(
+  image="fig19_lighthouse_audit_offscreen.png",
+  alt="Audit <span lang=\"en\">Lighthouse</span>&nbsp;: Hors-Écran.",
```

Comment:
We don't use HTML tags in alt attributes as some dispute whether they should be used or not. For screen-readers they should all be using the caption anyway (through `aria-labelledby`) not the alt.

```suggestion
  alt="Audit Lighthouse : Hors-Écran.",
```